### PR TITLE
OLM-High-OCP-21953-Ensure that operator deployment is in the master node

### PR DESCRIPTION
**Pull request by tbuskey**
Test Case [OCP-21953](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-21953) & part of Jira [OLM-1428](https://issues.redhat.com/browse/OLM-1428)

#### Does the following
- ` oc  get deployment -n openshift-marketplace marketplace-operator ` _to find if the spec is to be on the master node_
- `oc -n openshift-marketplace get pods -o wide` _to find the full name of the marketplace-operator pod_
  - find which node that pod is on
- look at the node to see if the label is set to be on the master node
- look at the node to see if the pod is actually on the master node

#### How to test:
`./extended-platform-tests run all --dry-run | grep 'Ensure that operator deployment is in the master node' | ./extended-platform-tests run -f - `

#### Successful run:

> Mar  2 15:22:59.096: INFO: The node ip-10-0-157-134.us-east-2.compute.internal of marketplace-operator-7cb9b6b76f-s2gnr pod is on the master node
> 
> passed: (7.9s) 2020-03-02T20:22:59 "[Feature:Platform] an end user use OLM Ensure that operator deployment is in the master node [Suite:openshift/conformance/parallel]"
> 1 pass, 0 skip (7.9s)

> oc version
> Client Version: openshift-clients-4.3-31-gc25fb9c7
> Server Version: 4.3.0-0.nightly-2020-03-02-094404
> Kubernetes Version: v1.16.2
> 
[testrun-21953.log](https://github.com/openshift/openshift-tests/files/4277812/testrun-21953.log)

### CODE REVIEW

**Comment by tbuskey** [comment]
@jianzhangbjz @bandrade @scolange @kuiwang02 @mffiedler

File: test/extended/operators/olm.go [hunk 1]
```diff
@@ -251,6 +251,88 @@ var _ = g.Describe("[Feature:Platform] an end user use OLM", func() {
 		}
 	})
 
+	// author: tbuskey@redhat.com
+	g.It("OLM-Low-OCP-21953-Ensure that operator deployment is in the master node", func() {
```
Comment (jianzhangbjz) [inline test/extended/operators/olm.go]: Here the importance of OCP-21953 should be `High`, not `Low`.

**Comment by tbuskey** [comment]
moved to private branch

**Pull request closed without merging**